qutebrowser/completion/completiondelegate.py

```diff
@@ -198,8 +198,8 @@ def _get_textdoc(self, index):
             columns_to_filter = index.model().srcmodel.columns_to_filter
             if index.column() in columns_to_filter and pattern:
                 repl = r'<span class="highlight">\g<0></span>'
-                text = re.sub(re.escape(pattern), repl, self._opt.text,
-                              flags=re.IGNORECASE)
+                text = re.sub(re.escape(pattern).replace(r'\ ', r'|'),
+                              repl, self._opt.text, flags=re.IGNORECASE)
```

Comment:
Currently with an URL like http://qutebrowser.org:8010/waterfall?reload=5, when entering "qute t", all the t's are highlighted, i.e. h**tt**p://qu**t**ebrowser.org:8010/wa**t**erfall?reload=5.

That once accidentally made me press e.g. `eb` first before realizing my mistake. But I'm not sure if there's an easy way to only highlight the correct occurence - can you think of one?